proposals/incubation/strimzi.md

```diff
@@ -0,0 +1,123 @@
+
+In addition, Strimzi is used by [Knative][knative] to provide its Kafka source.
+
+Strimzi can be installed using [Helm][helm] charts or using [OperatorHub.io][operatorhub] which is part of the [Operator Framework][operatorframework] project.
+
+Strimzi joined CNCF Sandbox in September 2019
+
+## Statement on alignment with the CNCF mission
+
+> The Foundation's mission is to make cloud native computing ubiquitous
+
+While Kafka is _capable_ of horizontal scalability, achieving such things as zero-downtime upgrades, and scaling clusters within Kubernetes without an operator is too high a barrier for many organizations.
+This prevents ubiquitous use of Kafka anywhere within the cloud.
+
+Strimzi provides declarative APIs for Apache Kafka resources, including broker clusters, Kafka Connect clusters, connectors, and Kafka MirrorMaker instances (for inter-cluster replication).
+
+By automating the operational burdens of running Kafka, Strimzi brings this important technology into the hands of anyone using Kubernetes.
+
+
+## Community
+
+In the Strimzi community users and developers can interact in several places: 
+* A public channel on the CNCF slack and mailing list
+* Github issues and discussions
+* A public "community call" every other week
+* We have participated in numerous KubeCon "Office Hours" events.
+
+Strimzi encourages contributions from the community, and has a good record of merging such contributions.
+
+
+## Incubation Stage Requirements
+
+<!-- NOTE: 
+See [Incubation Stage](https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubating-stage) requirements. -->
+
+### Production usage
+
+Strimzi is used in production by a number of organizations (see [ADOPTERS.md](https://github.com/strimzi/strimzi-kafka-operator/blob/main/ADOPTERS.md) for some public references). 
+In addition, the maintainers are aware of further non-public users of both Strimzi and commercially supported distributions of Strimzi. 
+
+### Committers
+
+Strimzi currently has 7 maintainers, who have commit rights.
+The [governance](https://github.com/strimzi/governance) model also defines area-specific owners, whose reviews count towards PR approval.
+
+Those maintainers are currently employed by three separate companies: Red Hat, IBM and CN Group.
+
+### Commits and merged contributions
+
+At the time of writing:
+* 2316 commits in the last 12 months, and more in the last 3 months than the previous 3 months
+* 438 contributors in the last 12 months
+* more than 3k stars and ~830 forks on github.
+
+See our [devstats](https://all.devstats.cncf.io/d/54/project-health-table?orgId=1&var-repogroup_name=Strimzi) for more details.
+
+### Versioning scheme
+
+The project's release version in described in [RELEASES.md](https://github.com/strimzi/governance/blob/main/RELEASES.md).
+
+### Security processes
+
+The project's security processes are described in [SECURITY.md](https://github.com/strimzi/.github/blob/main/SECURITY.md).
```

Comment:
Again, maybe you can add some more stuff here ... at least some general things how committed to security we are and how more details can  be found on the link?

Reply:
I just took a couple of sentences from the linked file.